pkg/resources/samples/gadget.kinvolk.io_v1alpha1_trace.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   node: minikube
   gadget: dns
   filter:
-    namespace: kube-system
+    namespace: gadget
     podname: etcd-minikube
```

Comment:
This change can be removed too since etcd does not live in the gadget namespace.

Ditto in pkg/resources/samples/trace-dns.yaml

Reply:
I need them otherwise the CI fails:
https://github.com/kinvolk/inspektor-gadget/runs/4525048090?check_suite_focus=true#step:10:41